src/Umbraco.Web/PropertyEditors/GridPropertyEditor.cs

```diff
@@ -81,11 +98,10 @@ public override object FromEditor(ContentPropertyData editorValue, object curren
                 {
                     // Parse the HTML
                     var html = rte.Value?.ToString();
-                    
-                    var userId = Current.UmbracoContext.Security.CurrentUser.Id;
 
-                    // TODO: In future task(get the parent folder from this config) to save the media into
-                    var parsedHtml = TemplateUtilities.FindAndPersistPastedTempImages(html, Constants.System.Root, userId, _mediaService, _contentTypeBaseServiceProvider);
+                    var userId = _umbracoContextAccessor.UmbracoContext?.Security.CurrentUser.Id ?? -1;
```

Comment:
Don't use magic integers.. We have a constant for this `-1`

```suggestion
                    var userId = _umbracoContextAccessor.UmbracoContext?.Security.CurrentUser.Id ?? Constants.Security.SuperUserId;
```